socorro/external/es/crashstorage.py

```diff
@@ -168,13 +184,39 @@ def reconstitute_datetimes(processed_crash):
                 # not there? we don't care
                 pass
 
+    @staticmethod
+    def remove_bad_keys(raw_crash):
+        """Removes keys from the top-level of the dict that are bad
+
+        Good keys satisfy the following properties:
+
+        * have one or more characters
+        * are composed of a-zA-Z0-9_-
+
+        Anything else is a bad key and needs to be removed.
+
+        This modifies the crash in-place.
+
+        :arg dict raw_crash: raw crash data
+
+        """
+        if not raw_crash:
+            return
```

Comment:
The processor is a crazy place and defensive programming seems like a good thing.